src/scripts/Battle.ts

```diff
@@ -115,9 +115,9 @@ class Battle {
         if (enemyPokemon.shiny) {
             GameHelper.incrementObservable(App.game.statistics.shinyPokemonEncountered[enemyPokemon.id]);
             GameHelper.incrementObservable(App.game.statistics.totalShinyPokemonEncountered);
-            App.game.logbook.newLog(LogBookTypes.SHINY, `[${Routes.getRoute(player.region, player.route()).routeName}] You encountered a wild shiny ${enemyPokemon.name}.`);
+            App.game.logbook.newLog(LogBookTypes.SHINY, `[${Routes.getRoute(player.region, player.route()).routeName}] You encountered a wild shiny ${enemyPokemon.displayName}.`);
```

Comment:
Yep, more complicated log entry object was needed to do this properly. I figured this was the best option for now until a proper solution

Reply:
We will also need to encode these strings as otherwise the save will fail to download when it contains certain characters:
![image](https://user-images.githubusercontent.com/7288322/172749684-a84e5f86-475d-4992-bd67-43781314a31e.png)

Reply:
I think I'll just do the slightly more complicated log entries for this, it will make for a good example of translation content with variables